src/gluonts/dataset/split.py

```diff
@@ -328,11 +328,11 @@ def test_pair(
 
 
 @dataclass
-class TestDataset:
+class TestData:
```

Comment:
Should we mark this as breaking?

Reply:
it's only breaking wrt `dev` itself, since this code was merged after the `v0.10.0` release